src/Honeybee_Select by Type.py

```diff
@@ -61,25 +64,36 @@ def main(HBZones):
     
     # call the objects from the lib
     hb_hive = sc.sticky["honeybee_Hive"]()
-    HBZoneObjects = hb_hive.callFromHoneybeeHive(HBZones)
+    HBZoneObjects = hb_hive.callFromHoneybeeHive(_HBZones)
     
     for zone in HBZoneObjects:
+        
+        # Extract surfaces from HBZones?
         for srf in zone.surfaces:
             try:
                 if srf.type in showCases:
                     if srf.type == 0 and srf.hasChild:
                         surfaces.append(srf.punchedGeometry)
+                        
+                        HBSurfacestypezero.append(srf) # Added by Anton Szilasi for Honeybee PV generator component
+                        
                     else:
                         surfaces.append(srf.geometry)
-                        
+
+                        otherHBSurfaces.append(srf) # Added by Anton Szilasi for Honeybee PV generator component
```

Comment:
@antonszilasi why don't you ad a new output just for PV? I would name it explicitly after the type. otherHBSurfaces can be really ambiguous.